app/models/case/base.rb

```diff
@@ -931,6 +932,15 @@ def validate_received_date
     errors[:received_date].any?
   end
 
+  def validate_email_format
+    if email =~ /\A.+@.+\z/ && email.present?
```

Comment:
just being very picky 😳 ,  it maybe better write as `email.present? && email =~ /\A.+@.+\z/`?

Reply:
Yes that would be better I'll make that change! :)